packages/core/src/util/esm.js

```diff
@@ -66,3 +66,31 @@ exports.isESMApp = function isESMApp() {
 
   return isESM;
 };
+
+exports.tracerInstrumentationInfo = function tracerInstrumentationInfo() {
+  const instrumentationMap = {
```

Comment:
This logic should not be here, this file specific to esm.

Reply:
Okay will update.